### PR TITLE
rewrite e2e

### PR DESCRIPTION
**Pull request by xiaojingchen**
this pr rewrite e2e test and refactor the test code. the test contains e2e test and stability test.

### CODE REVIEW

File: tests/actions.go [hunk 1]
```diff
@@ -315,6 +341,12 @@ func (oa *operatorActions) CleanTidbCluster(info *TidbClusterInfo) error {
 		}
 	}
 
+	// delete all jobs
+	allJobsSet := label.Label{}.Instance(info.ClusterName).String()
```
Comment (weekface) [inline tests/actions.go]:
```suggestion
	allJobsSet := label.New().Instance(info.ClusterName).String()
```

File: tests/manifests/e2e/e2e.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+spec:
+  serviceAccount: tidb-operator-e2e
+  containers:
+  - name: tidb-operator-e2e
+    image: ""
+    imagePullPolicy: Always
+    command:
+    - /usr/local/bin/e2e
+    - --operator-tag=e2e
+    - --operator-image=pingcap/tidb-operator:latest
```
Comment (weekface) [inline tests/manifests/e2e/e2e.yaml]:
```suggestion
    - --operator-image=localhost:5000/pingcap/tidb-operator:latest
```

File: tests/actions.go (hunk 1 above)
Comment (xiaojingchen) [reply tests/actions.go]: the namespace contains some jobs which label `app.kubernetes.io/managed-by: Tiller`.
these jobs cannot be select by `label.New()`

**Pull request merged**